src/megaclient.cpp

```diff
@@ -1340,9 +1375,31 @@ void MegaClient::exec()
                                 fetchnodes(true);
                                 reqtag = creqtag;
                             }
+                            else if (e == API_EAGAIN || e == API_ERATELIMIT)
+                            {
+                                if (!statecurrent)
```

Comment:
`pendingsc` is the channel to receive action packets. `fetchingnodes` means that a fetchnodes command is being sent (for that reason it's used in `pendingcs`), but action packets can be required for the initialization of the filesystem even if a fetchnodes command is not sent (when there is a local cache). The intention is to measure the number of action packets processed before the filesystem is current, so the `statecurrent` flag is perfect for that purpose in all cases.